content/en/docs/concepts/services-networking/service.md

```diff
@@ -235,19 +236,19 @@ By default, kube-proxy in userspace mode chooses a backend via a round-robin alg
 
 In this mode, kube-proxy watches the Kubernetes control plane for the addition and
 removal of Service and Endpoint objects. For each Service, it installs
-iptables rules which capture traffic to the Service's `clusterIP` (which is
-virtual) and `port` and redirects that traffic to one of the Service's
+iptables rules, which capture traffic to the Service's `clusterIP` and `port`,
+and redirects that traffic to one of the Service's
 backend sets.  For each Endpoint object, it installs iptables rules which
 select a backend Pod.
 
 By default, kube-proxy in iptables mode chooses a backend at random.
 
 Using iptables to handle traffic has a lower system overhead, because traffic
-is handled by Linux netfilter without the need switch between userspace and the
+is handled by Linux netfilter. It removes the need to switch between userspace and the
 kernel space. This approach is also likely to be more reliable.
```

Comment:
What is the `it`, iptables or Linux `netfilter`?

Reply:
true.. That's confusing.. am reverting back to the old sentence and did a small nit there.